statefulset.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: hazelcast
+spec:
+  replicas: 3
+  serviceName: hazelcast
+  selector:
+    matchLabels:
+      app: hazelcast
+  template:
+    metadata:
+      labels:
+        app: hazelcast
+    spec:
+      containers:
+        - name: hazelcast
+          image: hazelcast/hazelcast-enterprise:4.0
```

Comment:
Can we update this image version to `4.1`?

And the same for all other k8 guides?

Reply:
Changed here, will update other k8s guides